### ISSUE TITLE
YuppTv site isn't. Loading

### ISSUE THREAD

**Issue by phpsieve** [SxtBox/Yupptv_Dot_Com_Streaming#2]
Hello sir , I looked upon your api for YuppTv and found it great but I wonder how to run those links as I wanted to try those once . Those show http code failure .

**Comment by SxtBox** [comment]
Hello, are currently in demo mode, if you want to try it live mode sendme msg to send links.

**Comment by phpsieve** [comment]
Yes sir i wanted to try it but is it having paid channel also if yes then i
want to try it

**Comment by SxtBox** [comment]
Nope, free ch only!
![Screenshot_ 2021-02-22 _ 16-32-34](https://user-images.githubusercontent.com/60861585/108730387-c55a3900-752b-11eb-97be-236492621b2a.png)
